SoftLayer/CLI/routes.py

```diff
@@ -57,6 +57,7 @@
     ('virtual:notifications', 'SoftLayer.CLI.virt.notifications:cli'),
     ('virtual:notification-add', 'SoftLayer.CLI.virt.notification_add:cli'),
     ('virtual:notification-delete', 'SoftLayer.CLI.virt.notification_delete:cli'),
+    ('virtual:host-list', 'SoftLayer.CLI.virt.host_list:cli'),
```

Comment:
We already have a way to list dedicated hosts, so I would prefer we use that single command. I don't mind there being an alias for it for `slcli vs host-list` though. I think this should work to use the same code for 2 commands.
```suggestion
    ('virtual:host-list', 'SoftLayer.CLI.dedicatedhost.list:cli'),
```

Reply:
Done